src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CallbackResultControllerTest.java

```diff
@@ -0,0 +1,230 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.data.callbackresult.CallbackResult;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CallbackResultService;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.Calendar;
+import java.util.TimeZone;
+import java.util.UUID;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.hamcrest.Matchers.hasSize;
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.data.callbackresult.RequestType.ATTACH_TO_CASE;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.data.callbackresult.RequestType.CREATE_CASE;
+
+@ExtendWith(SpringExtension.class)
+@WebMvcTest(CallbackResultController.class)
+public class CallbackResultControllerTest {
+
+    private static final String CASE_ID1 = "CASE_ID1";
+    private static final String CASE_ID2 = "CASE_ID2";
+    private static final String EXCEPTION_RECORD_ID1 = "ER_ID1";
+    private static final String EXCEPTION_RECORD_ID2 = "ER_ID2";
+    private static final String CREATED_AT2 = "2021-05-05T01:36:22.727Z";
+    private static final String CREATED_AT1 = "2021-05-05T01:35:22.727Z";
+    private static final Instant TIMESTAMP1;
+    private static final Instant TIMESTAMP2;
+
+    static {
+        TimeZone timeZone = TimeZone.getTimeZone(ZoneId.of("Europe/London"));
+        Calendar cal = Calendar.getInstance(timeZone);
+        cal.set(Calendar.YEAR, 2021);
+        cal.set(Calendar.MONTH, 4);
+        cal.set(Calendar.DAY_OF_MONTH, 4);
+        cal.set(Calendar.HOUR, 14);
+        cal.set(Calendar.MINUTE, 35);
+        cal.set(Calendar.SECOND, 22);
+        cal.set(Calendar.MILLISECOND, 727);
+        TIMESTAMP1 = Instant.ofEpochMilli(cal.getTimeInMillis());
```

Comment:
is this not work :
Instant.parse("2021-04-30T18:35:24.00Z");  ?
you can add zone as well.